### ISSUE TITLE
Allow 'svg' as a descendant of of tag 'amp-nested-menu'. 

### ISSUE THREAD

**Issue by erakor** [ampproject/amphtml#26463]
Currently, the documentation only provides [one example of "Disallowed tag ancestor" error](https://amp.dev/documentation/guides-and-tutorials/learn/validation-workflow/validation_errors/#disallowed-tag-ancestor):
> Currently, the only example is a `template` tag, which may not be nested under another `template` tag.

However, it seems that the [`svg` tag is cannot be nested under `amp-nested-menu`](https://github.com/ampproject/amphtml/blob/master/extensions/amp-nested-menu/validator-amp-nested-menu.protoascii#L125) either. 
Could it be added?

If not, we should document that "The tag 'svg' may not appear as a descendant of tag 'amp-nested-menu'."
